xbmc/input/XBMC_keysym.h

```diff
@@ -222,15 +222,18 @@ typedef enum {
   // Add any other keys here
 
 	/* Media keys */
-  XBMCK_EJECT             = 333,
   XBMCK_STOP              = 337,
   XBMCK_RECORD            = 338,
   XBMCK_REWIND            = 339,
   XBMCK_PHONE             = 340,
   XBMCK_PLAY              = 341,
   XBMCK_SHUFFLE           = 342,
   XBMCK_FASTFORWARD       = 343,
+  XBMCK_EJECT             = 344,
```

Comment:
> Do not use whitespace to vertically align around operators or values. This causes problems on code review if one needs to realign all values to their new position, producing unnecessarily large diffs.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#35-vertical-alignment

Reply:
Sorry but all the file aligned with these whitespaces, if I follow the guide here, the file would be not consistent. Should I?

Reply:
Also please note in this context it's important that the values are aligned.

Reply:
ping

Reply:
Yes please follow the current code guide line for new or changed lines.